### DISCUSSION TITLE
Would you be able to implement navigating through Manager interface using a controller?

### DISCUSSION THREAD

**Discussion by Bobble80** [xenia-manager/xenia-manager#168] [Ideas]
Hi, I was wondering if you could look into adding the option to navigate through Xenia Manager interface using a controller?

I use a lot of emulators on my ROG Ally and a couple of them allow you to navigate them using the built in controller to both change settings and go through your game library and start games. 
I know PPSSPP, RetroArch, Yuzu and RPCS3 all have this feature and I think a few others like Duckstation and PCSX2 have when in their big picture modes. 

Would you be able to look into adding this into Xenia Manager at some point?

**Comment by shazzaam7** [comment]
Yes, that's one of the reasons for the big rewrite for v2.0.0 because I want to create something similar to Steam's Big Picture Mode (because I will need to use different framework for the Big Picture Mode since the one I'm using doesn't support controllers at all)
